ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -325,6 +325,18 @@ ORK_CLASS_AVAILABLE
  */
 @property (readonly, nullable) NSString *minimumValueDescription;
 
+/**
+ An image for the upper bound of the slider. Recommendation image size is 30px by 30px.
```

Comment:
I think rewriting as "*The recommended image size is 30 x 30 points.*" would be more in Apple style, and more technically correct (resolution independent images).